frontend/src/features/ratings/RatingOrderByInput.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import {
+  Box,
+  FormControl,
+  InputLabel,
+  ListItemText,
+  MenuItem,
+  Select,
+  SelectChangeEvent,
+} from '@mui/material';
+import CallMadeIcon from '@mui/icons-material/CallMade';
+import CallReceivedIcon from '@mui/icons-material/CallReceived';
+
+import { TitledSection } from 'src/components';
+import { useCurrentPoll } from 'src/hooks';
+import { getMetadataName } from 'src/utils/constants';
+
+interface FilterProps {
+  value: string;
+  onChange: (value: string) => void;
+}
+
+function RatingOrderByInput(props: FilterProps) {
+  const { t } = useTranslation();
+  const { name: pollName, options } = useCurrentPoll();
+
+  const extraMetadataOrderBy = options?.extraMetadataOrderBy ?? [];
+
+  const handleChange = (event: SelectChangeEvent) => {
+    props.onChange(event.target.value);
+  };
+
+  const genericItems = useMemo(
+    () => [
+      {
+        label: t('ratingOrderByInput.lastComparisonDate'),
+        value: 'last_compared_at',
+        icon: CallMadeIcon,
+      },
+      {
+        label: t('ratingOrderByInput.lastComparisonDate'),
+        value: '-last_compared_at',
+        icon: CallReceivedIcon,
+      },
+      {
+        label: t('ratingOrderByInput.lastComparisonDate'),
+        value: 'n_comparisons',
+        icon: CallMadeIcon,
+      },
+      {
+        label: t('ratingOrderByInput.lastComparisonDate'),
+        value: '-n_comparisons',
+        icon: CallReceivedIcon,
```

Comment:
The same label is used 4 times.

Reply:
Fixed by https://github.com/tournesol-app/tournesol/pull/1181/commits/3be9baf39d85d851ae5e95a0ebb80981be6ef2d9